src/pvecontrol/config.py

```diff
@@ -10,16 +11,30 @@
             "host": str,
             "user": str,
             "password": str,
+            "timeout": int,
```

Comment:
Timeout must be optional, and global I think. not by cluster.

Actually this crash with a template error if not present in configuration.